packages/tools/devtools/devtools-view/src/components/UnknownDataView.tsx

```diff
@@ -18,5 +21,13 @@ export interface UnknownDataViewProps {
 export function UnknownDataView(props: UnknownDataViewProps): React.ReactElement {
 	const { node } = props;
 
-	return <div>{`Encountered an unrecognized kind of data object: ${node.nodeKind}`} </div>;
+	return (
+		<TreeItem v-bind:leaf="true">
```

Comment:
This isn't a Vue directive, is it? lol Can you explain what this does? I tried looking through the docs and might have missed it, but didn't see any v-bind

Reply:
😊Not vue, it's used for binding a boolean value which is a prop for `TreeItem`.

Reply:
Curious to see what the before and after look like for this. If possible I'd rather not load from an internal module of an external dependency, especially one called `unstable` 😄 .

Reply:
I would also prefer to keep the `Tree` related components out of these data views if possible. These already get wrapped in a TreeItem (or they should be if not) by an ancestor component. These views should be unopinionated about what kind of structure they get rendered into.

Ideally we could fix these styling changes by modifying `Tree.tsx`.

We may need to not use `div` in these data views though.

Reply:
#### PR 
https://github.com/microsoft/FluidFramework/pull/15172#discussion_r1177146308

As @Josmithr mentioned, `UnknownObjectView` and `UnknownFluidObjectView` are wrapped with `<div>` tag since the item is already wrapped by `<TreeItem>` by its ancestor.

Reply:
That would be a good test case to add to our app, though. I'll add it :)

Reply:
PR: #15557

Reply:
I also have a PR up doing some cleanup with the tree factoring. The FluentUI Tree library is _very_ strange. After banging my head against it for a while, I finally figured out what they are trying to do, and updated our code to line up with their intentions better.

#15569